app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,23 @@
     <string name="rate">RATE APP</string>
```

Comment:
What I meant is that the string "RATE APP" I use it as an option of a menu, and when I write the xml of that menu I don't have the property that I mentioned above, so in order to use the string in that way I have to write all in caps in the string.xml file.